tests/io/test_binary.cpp

```diff
@@ -26,7 +26,7 @@
 
```

Comment:
I am disabling the old io tests with `tml_full` data. They don't work well anymore because the column of hit_id has been changed to `measurement_id` in recent data.

This is the error message that I got when I read the cell data from `tml_full`
`C++ exception with description "Missing header column 'measurement_id'" thrown in the test body.`